dashboard/prepare-dashboard-data.py

```diff
@@ -289,6 +290,15 @@ def count_dups(hvr_fname):
         sample_metadata[sample]["reads"] = \
             project_sample_reads[project][sample]
 
+        rc_fname = "ribocounts/%s.ribocounts.txt" % sample
+        try:
+            with open(rc_fname, 'r') as file:
+                content = file.read().strip()
+                ribocount = int(content)
+            sample_metadata[sample]["ribocounts"] = ribocount
+        except FileNotFoundError:
+            continue
```

Comment:
I'd like `pass` here better than `continue`.  If someone later adds code below that's supposed to be within this loop then we wouldn't want to skip it in cases where there was no ribocounts file.